src/v/redpanda/admin_server.cc

```diff
@@ -121,6 +127,9 @@ void admin_server::configure_admin_routes() {
     register_status_routes();
     register_broker_routes();
     register_partition_routes();
+    if (finjector::shard_local_badger().is_enabled()) {
+        register_hbadger_routes();
```

Comment:
we should either (1) register a GET route at `v1/failure-probes` which returns a flag indicating if the probes are enabled or not or (2) register all the routes but have them return an appropriate http error code when things are disabled.

by not registering the routes we'll send the tests a 404 which is hard to work with in the context of a feature that can be enabled/disabled because its the same code used for so many other common mistakes, typos, etc..

Reply:
good point.

Reply:
done